ClinGen/clincoded
# Connects to #1639. Associate users with affiliations.

**Steps to test:**
1. Go to ^/users page and filter for the users `Rebecca Siegert` and `Andrew Grant`.
2. Expect to see these 2 users are now associated with **10005** and **10006** affiliations.
